xilinx/common/utils/.gdb_history

```diff
@@ -0,0 +1,2 @@
+exit()
+file prjxray_tile_import.py
```

Comment:
Is this file needed? Was it commited by accident?

Reply:
By mistake!